lib/simulator-xcode-6.js

```diff
@@ -306,10 +293,10 @@ class SimulatorXcode6 {
     log.debug('Attempting to launch and quit the simulator, to create directory structure');
     log.debug(`Will launch with Safari? ${safari}`);
 
+    await this.run();
+
     if (safari) {
-      await instrumentsUtil.quickLaunch(this.udid, 'com.apple.mobilesafari');
-    } else {
-      await this.run();
+      await this.openUrl('http://www.appium.io');
```

Comment:
Hmm. Except this repo is standalone, and doesn't depend on appium.
It's ok if safari can't load the url, safari will still open and will just show "safari was not able to load the page"